pkgs/racket-doc/scribblings/reference/logging.scrbl

```diff
@@ -142,10 +144,14 @@ Defines @racketkeywordfont{log-}@racket[id]@racketkeywordfont{-fatal},
 like @racket[log-fatal], @racket[log-error],@racket[log-warning],
 @racket[log-info], and @racket[log-debug]. The @racket[define-logger]
 form also defines @racket[id]@racketidfont{-logger}, which is a logger with
-default topic @racket['@#,racket[id]] that is a child of @racket[(current-logger)];
-the @racketkeywordfont{log-}@racket[id]@racketkeywordfont{-fatal},
-@|etc| forms use this new logger. The new logger is
-created when @racket[define-logger] is evaluated.}
+default topic @racket['@#,racket[id]] that is a child of the result of
+@racket[parent-expr], or of @racket[(current-logger)] if
+@racket[parent-expr] not provided; the
```

Comment:
`parent-expr` might be not provided or `#false` now

Reply:
`#f` means a different thing from specifying `(current-logger)`… do you think it needs to be reworded somehow, given that?

Reply:
Oh, I misunderstood what `#f` means.

But yes I think it should be reworded. Maybe add a parenthetical like the one for `make-logger`, something like:

> .... that is a child of the result of `parent-expr` (when `parent-expr` is not `#f`), or of ....

Or maybe point to `make-logger` (this suggestion doesn't sound as good):

> The `define-logger` form also defines a new logger via `make-logger`; if `parent-expr` is not provided then `(current-logger)` is used.

Reply:
I’ve updated the documentation to include your first suggestion.